### ISSUE TITLE
Max slots don't work as expected

### ISSUE THREAD

**Issue by PlatinMTA** [multitheftauto/mtasa-blue#1503]
**Describe the bug**
If you have on mtaserver.conf the maxplayers setting set to, for example, 50, but the config setMaxPlayers() set to 25, is still possible to players to bypass this config and make the server have 26/25 players or more.

**To reproduce**
maxplayers 2 in mtaserver.conf
setMaxPlayers set to 1
join two players at the same time

**Expected behaviour**
Players should not join if there are not more slots available.

**Screenshot**
![image](https://user-images.githubusercontent.com/23291573/84100919-3a5c0180-a9e3-11ea-90f2-a0481bdc4723.png)

**Version**
Server: r20503 (linux)

**Additional context**
I really never tested if changing the maxplayers to 25 would solve the problems, but seeing that maxplayers allocates memory maybe it fixes it.

If someone from my discord has an screenshot i will add it here. I can't find any.

**Comment by qaisjp** [comment]
I don't think this is related to the max players Lua function / server config setting - I think this is just what happens when two players join at the same time. But this is just a hunch.

**Comment by PlatinMTA** [comment]
> I don't think this is related to the max players Lua function / server config setting - I think this is just what happens when two players join at the same time. But this is just a hunch.

Yeah it could just be that and everything else doesn't really matter to this issue. But seeing that, afaik, maxplayers (from mtaserver.conf) allocates some memory, maybe it was related to that.

If I wasnt so lazy I would test it, but w/e it isn't a major issue anyway.
